doc/release-notes-17410.md

```diff
@@ -0,0 +1,4 @@
+Command-line options
+--------------------
+
+- The `-debug=db` logging category has been renamed to `-debug=walletdb`, to distinguish it from `coindb`.
```

Comment:
```suggestion
- The `-debug=db` logging category has been renamed to `-debug=walletdb`, to distinguish it from `coindb`.
   `-debug=db` has been deprecated and will be removed in the next major release.
```

Reply:
Thanks, will add that.
Was trying to keep it succinct, this is not worth making the release notes much longer over. But that's good to mention.